src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -8885,12 +8885,13 @@ int wm_vuldet_get_software(int agent_id, bool not_triaged, cJSON** requested_ite
 
 void wm_vuldet_init_queues() {
     vu_tasks_queue = linked_queue_init(vu_tasks_queue_push_callback, vu_tasks_queue_pop_callback);
-    vu_messages_queue = linked_queue_init(NULL, NULL);
+    vu_messages_queue = queue_init(getDefine_Int("wazuh_vuln_detector", "task_message_queue_size",
```

Comment:
I think the read of the internal options file should occur outside this method. For example, at the beginning of the vulnerability detector main.

Reply:
Moved!